src/input/source/led.rs

```diff
@@ -0,0 +1,98 @@
+pub mod multicolor_chassis;
+use std::error::Error;
+//use glob_match::glob_match;
+use self::multicolor_chassis::MultiColorChassis;
+use super::{SourceDeviceCompatible, SourceDriver};
+use crate::{
+    config, constants::BUS_SOURCES_PREFIX, input::composite_device::client::CompositeDeviceClient,
+    udev::device::UdevDevice,
+};
+/// List of available drivers
+enum DriverType {
+    MultiColorChassis,
+}
+/// [LedDevice] represents an input device using the leds subsystem.
+#[derive(Debug)]
+pub enum LedDevice {
+    MultiColorChassis(SourceDriver<MultiColorChassis>),
```

Comment:
We might consider changing this class name fro MultiColorChassis to JoystickRing since that's what the new asus driver calls them and I'll be changing ayn/ayaneo-platform drivers to do the same.